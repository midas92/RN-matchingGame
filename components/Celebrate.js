import React from 'react'
import ConfettiCannon from 'react-native-confetti-cannon';

const Celebrate = ({ fire }) => {
  if(fire) {
    return(
      <>
        <ConfettiCannon
          count={100}
          origin={{x: 0, y: 0}}
          fadeOut={true}
          explosionSpeed={500}
        />
        <ConfettiCannon
          count={100}
          origin={{x: 200, y: 0}}
          fadeOut={true}
          explosionSpeed={500}
        />
        <ConfettiCannon
          count={100}
          origin={{x: 400, y: 0}}
          fadeOut={true}
          explosionSpeed={500}
        />
      </>
    );
  } else {
    return null;
  }
}

export default Celebrate;
