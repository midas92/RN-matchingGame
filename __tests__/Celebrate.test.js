import React from 'react';
import { render } from '@testing-library/react-native';
import Celebrate from '../components/Celebrate';

describe('<Celebrate/>', () => {
  it('should match snapshot', () => {
    const result = render(<Celebrate/>).toJSON();
    expect(result).toMatchSnapshot();
  });

  it('should render the Celebrate component', () => {
    const result = render(<Celebrate/>);
    expect(result).toBeTruthy();
  });
});
