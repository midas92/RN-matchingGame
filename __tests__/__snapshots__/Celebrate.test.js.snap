// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Celebrate/> should match snapshot 1`] = `null`;
